sdcm/nemesis.py

```diff
@@ -481,6 +482,40 @@ def disrupt_rolling_restart_cluster(self, random=False):
         self._set_current_disruption('RollingRestartCluster %s' % self.target_node)
         self.cluster.restart_scylla(random=random)
 
+    def disrupt_toggle_authenticator_type(self):
+        """
+        If prepare_saslauthd is enabled, saslauthd and ldap environment will be prepared for
+        using SaslauthdAuthenticator. We have same account (cassandra) for SaslauthdAuthenticator
+        and PasswordAuthenticator, so it can be toggled smoothly without effect to c-s workloads.
+
+        It's only support to switch between PasswordAuthenticator and SaslauthdAuthenticator,
+        the authenticator will be reset back in the end of nemesis.
+        """
```

Comment:
What I suggest to do here is change the nemesis to be "switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_back".

The nemesis will switch the authenticator, wait for few minutes and then switch back to the original authenticator.
This way we make sure that a jobs of SaslauthdAuthenticator will actually use this authenticator for the entire job.

Reply:
fixed.